### ISSUE TITLE
Training advice and outcomes

### ISSUE THREAD

**Issue by GeEom** [davidbrowne17/csm-streaming#9]
I am hoping to finetune the model onto a specific voice (my voice). Is this something that can be trained in such that context isn't necessary?

I trained via the default lora script on twenty minutes of single speaker data. I was hoping the model would pick up the speaker likeness without context. So far no luck. I might manually transcribe my audio_data as the automatic hasn't been very accurate.

My other thought is I may be under trained? What order of magnitude is correct? Should I be doing 5 epochs, 50, 500? Any steer really appreciated!

**Comment by davidbrowne17** [comment]
Usually only 1-5 epochs is needed. You will probably still need context for good results. Try using the develop branch, this branch creates dynamic PNG images graphing your training run which will help give you an insight into how its going. I had good results on 5 epochs, 1e6 learning rate with 128 R and Alpha values. My dataset was about 40 mins long but had very clear audio clips.
